cmd/gpu_plugin/README.md

```diff
@@ -66,139 +56,37 @@ Intel GPU-plugin supports a few different operation modes. Depending on the work
 |:---- |:-------- |:------- |:------- |
 | shared-dev-num == 1 | No, 1 container per GPU | Workloads using all GPU capacity, e.g. AI training | Yes |
 | shared-dev-num > 1 | Yes, >1 containers per GPU | (Batch) workloads using only part of GPU resources, e.g. inference, media transcode/analytics, or CPU bound GPU workloads | No |
-| shared-dev-num > 1 && resource-management | Yes and no, 1>= containers per GPU | Any. For best results, all workloads should declare their expected GPU resource usage (memory, millicores). Requires [GAS](https://github.com/intel/platform-aware-scheduling/tree/master/gpu-aware-scheduling). See also [fractional use](#fractional-resources-details) | Yes. 1000 millicores = exclusive GPU usage. See note below. |
+| shared-dev-num > 1 && resource-management | Yes and no, 1>= containers per GPU | Any. For best results, all workloads should declare their expected GPU resource usage (memory, millicores). See also [use](./fractional.md) | Yes. 1000 millicores = exclusive GPU usage. See note below. |
```

Comment:
This table is somewhat confusing already. I'd like to decrease the information vs. adding more details. I think it would be enough to indicate that this use case requires more setup and a link to the instructions.

Reply:
What about:
```suggestion
| shared-dev-num > 1 && resource-management | Depends on resource requests | Any. For requirements and usage, see [fractional resource management](./fractional.md) | Yes. 1000 millicores = exclusive GPU usage. See note below. |
```
(This may require also some improvements to `fractional.md`, but that could come in separate PR.)